packages/vite/src/node/plugins/css.ts

```diff
@@ -246,6 +261,8 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
     extensions: [],
   })
 
+  const preprocessorWorkerController = createPreprocessorWorkerController()
```

Comment:
Should this be in a `buildStart` so it works in `build --watch`?

Reply:
Ah, forgot about that. Thanks!